README.md

```diff
@@ -6,7 +6,37 @@ and [OpenVSX](https://open-vsx.org/extension/redhat/ansible) compatible editors
 by leveraging
 [ansible-language-server](https://github.com/ansible/ansible-language-server).
 
-## Activating Red Hat Ansible extension
+## Language association to yaml files
+
+The extension works only when a document is assigned `ansible` language. The
+following method is used to assign `ansible` language to the document opened by
+the extension:
+
+### Without file inspection
+
+- yaml files under `/playbooks` dir.
+- files with the following double extension: `.ansible.yml` or `.ansible.yaml`.
+- notable yaml names recognized by ansible like `site.yml` or `site.yaml`
+- yaml files having playbook in their filename: `*playbook*.yml` or
+  `*playbook*.yaml`
+
+### With file inspection
+
+#### File inspection for ansible keywords
+
+- Primary method is inspection for top level playbook keywords like hosts and
+  import_playbook in yaml files.
+
+#### Modelines (optional)
+
+- If modelines present, it is given highest priority and language is set
```

Comment:
This sentence does not sound natural, can you please rephrase it and also include an example for selecting the language to yaml. Most people do not know what modelines are and how the look, so mentioning it would make it easier.

Ideally I would make the modelines keywork a link to an external side explaining them but I was not able to identify one that seem as a good one to use. Apparently wikipedia does not have a page about the subject, yet.

Reply:
Done. Rewrote the sentence, and provided links and examples as well.